config/was.yaml

```diff
@@ -130,21 +130,25 @@ input:
 output:
 
     nfiles: 1
-    dir: /lus/grand/projects/RomanDESC/roman_output/RomanWAS/images/truth
+    dir: RomanWAS/images/simple_model
     file_name:
         type: FormattedStr
-        format: "Roman_WAS_truth_%s_%i_%i.fits.gz"
+        format: "%s/%i/Roman_WAS_truth_%s_%i_%i.fits.gz"
         items:
+            - { type: ObSeqData, field: filter }
+            - { type: ObSeqData, field: visit }
```

Comment:
This breaks the assumption in https://github.com/DukeCosmology/roman_imsim/blob/main/roman_imsim/detector_physics.py#L130 that the output `file_name` has three items